ldap-api/src/main/java/ca/bc/gov/hlth/ldapapi/service/LdapService.java

```diff
@@ -181,13 +154,14 @@ protected void updateUserFailedLoginAttempts(String userInfoName) {
 
         } else { // Entry Exists, timestamp<1hr, attempts >=3
             lockUserAccount(userInfoName + "," + LDAP_SEARCH_BASE);
-            loginAttemptsForUser.setAttempts(currentAttempts + 1);
-            loginAttemptsForUser.setLastAttempt(LocalDateTime.now());
+            loginAttemptsMap.remove(userInfoName);
```

Comment:
This fixes a bug. When we lock the account, we want to reset attempts to zero. This is so that when the user gets their account unlocked by an admin, they don't immediately relock it after a single typo because attempts is already at 3. Instead of resetting it to zero, I just remove it from the map.